activetable/templates/html/activetable.html

```diff
@@ -0,0 +1,47 @@
+<div class="activetable_block">
+  {% if help_text %}
+  <div class="activetable-help" style="width: {{ total_width }}px;">
+    <button id="activetable-help-button">+help</button>
```

Comment:
The + and - are not ready by screen readers reliably.  This button should include an ```aria-expanded``` attribute that informs the user of the current state of the disclosure (help text).  There should also be an explicit reference to the associated ```<p>``` element with an ```aria-controls``` attribute and an ```aria-haspopup="true"``` attribute.

These IDs seem to be hard coded.  What happens if there are multiple active table problems on a single page?  IDs have to be unique, especially if they are being referenced by other elements, otherwise the whole reference system breaks down.